src/ensembl/src/shared/components/question-button/QuestionButton.tsx

```diff
@@ -30,43 +32,40 @@ export enum QuestionButtonOption {
 
 type Props = {
   helpText: React.ReactNode;
-  styleOption?: QuestionButtonOption;
+  styleOption: QuestionButtonOption;
   className?: { [key in QuestionButtonOption]?: string };
 };
 
 const QuestionButton = (props: Props) => {
-  const [isHovering, setIsHovering] = useState(false);
-  const elementRef = useRef<HTMLDivElement>(null);
+  const [hoverRef, isHovered] = useHover<HTMLDivElement>();
+  const [shouldShowTooltip, setShouldShowTooltip] = useState(isHovered);
```

Comment:
Would it not be possible to just use `isHovered` instead of having `shouldShowTooltip`?

Reply:
@msmanoj what I am trying to achieve here is:
- hide the tooltip when user removes mouse from the button
- hide the tooltip when user starts scrolling

`isHovered` doesn't help with the second point

Currently, the tooltip itself is aware of scrolling, and signals to the parent component that it wants to close when user starts scrolling. So I am combining the signals from the hook about whether user is mousing over the question button with the signal from the tooltip about whether user is scrolling. I could, of course, add a scroll listener to the QuestionButton itself, but I thought it better to use the one in the tooltip.

Reply:
@azangru do we need to hide the tooltip as soon as the user starts to scroll?

I just tried to scroll 1-2px but the tooltip goes away when I'm still hovered. Is this expected?

Reply:
I'd say yes. You usually don't scroll when you want to read a tooltip.